zerver/views/user_groups.py

```diff
@@ -119,8 +120,16 @@ def edit_user_group(
     can_mention_group_id: Optional[int] = REQ(
         "can_mention_group", json_validator=check_int, default=None
     ),
+    can_manage_group_id: Optional[Union[int, str]] = REQ(
+        "can_manage_group", json_validator=check_string_or_int, default=None
+    ),
 ) -> HttpResponse:
-    if name is None and description is None and can_mention_group_id is None:
+    if (
+        name is None
+        and description is None
+        and can_mention_group_id is None
+        and can_manage_group_id is None
```

Comment:
I remember that we discussed about removing this in some `#api design` discussion. I think we can do this as a follow-up.